integration/connection.tf

```diff
@@ -34,8 +34,8 @@ resource "materialize_connection_kafka" "kafka_conn_multiple_brokers" {
     database_name = materialize_secret.kafka_password.database_name
     schema_name   = materialize_secret.kafka_password.schema_name
   }
-  sasl_mechanisms = "SCRAM-SHA-256"
-  progress_topic  = "progress_topic"
+  sasl_mechanism = "SCRAM-SHA-256"
+  progress_topic = "progress_topic"
```

Comment:
I think that this is how we have it in the docs:

https://materialize.com/docs/sql/create-connection/#kafka-auth

It only accepts 1 parameter but is plural